### PR TITLE
Refactor backupstore codes

### PR DESCRIPTION
**Pull request by derekbit**
longhorn/longhorn#5189

### CODE REVIEW

File: backupstore.go [hunk 1]
```diff
@@ -66,7 +66,7 @@ func addVolume(volume *Volume, driver BackupStoreDriver) error {
 	}
 	log.Debug("Added backupstore volume ", volume.Name)
```
Comment (shuo-wu) [inline backupstore.go:67]: Redundant log?

**Review by shuo-wu — APPROVED** [review]
In general LGTM

File: backupstore.go (hunk 1 above)
Comment (derekbit) [reply backupstore.go]: Yeah. Rmoved. Thank you.

**Review by joshimoo — APPROVED** [review]
LGTM, thanks for pulling the refactoring changes out into their own PR

**Review by innobead — COMMENTED** [review]
This is another good example to refacotr when available. Take any chance if possible.

**Pull request merged**